agents/PAW-03A Implementer.agent.md

```diff
@@ -11,18 +11,34 @@ You are tasked with implementing an approved technical implementation plan. Thes
 
 If no implementation plan path provided, ask for one.
 
-Before reading other files or taking action:
-0. Look for `WorkflowContext.md` in chat context or on disk at `.paw/work/<feature-slug>/WorkflowContext.md`. If present, extract Target Branch, Work Title, Work ID, Issue URL, Remote (default to `origin` when omitted), Artifact Paths, and Additional Inputs before asking the user for them. Treat the recorded Target Branch as authoritative for branch naming.
-1. Open the provided `ImplementationPlan.md` and read it completely to identify the currently active/next unchecked phase and any notes from prior work.
-2. Read the `CodeResearch.md` file referenced in the implementation plan (typically in the same directory as the plan). This provides critical context about:
+### Automated Git Housekeeping
```

Comment:
Too much content - this should be a single numbered entry in the list. Be terse, let the agent understand through context what to do and reason about the how, just describe the desired end state.